ElementX/Sources/AppCoordinator.swift

```diff
@@ -97,6 +92,25 @@ class AppCoordinator: AuthenticationCoordinatorDelegate, Coordinator {
     
     // MARK: - Private
     
+    private func setupLogging() {
+        let loggerConfiguration = MXLogConfiguration()
+        
+        #if DEBUG
+        setupTracing(configuration: "info,hyper=warn,sled=warn,matrix_sdk_sled=warn")
+        loggerConfiguration.logLevel = .debug
+        #else
+        setupTracing(configuration: "info,hyper=warn,sled=warn,matrix_sdk_sled=warn")
+        loggerConfiguration.logLevel = .info
+        #endif
+        
+        // Redirect NSLogs to files only if we are not debugging
```

Comment:
Seeing as it took both of us a while to understand this:
```suggestion
        // Avoid redirecting NSLogs to files if we are attached to a debugger.
```